types/dat.gui/index.d.ts

```diff
@@ -108,14 +108,24 @@ export class GUI {
 }
 
 export class GUIController {
-    destroy(): void;
```

Comment:
@Teoxoy why did you remove the `destroy()` method? / The method still there from what I can see https://github.com/dataarts/dat.gui/blob/c2edd82e39d99e9d8b9f1e79011c675bc81eff52/src/dat/gui/GUI.js#L579
Could you please take a look to confirm?

Reply:
`destroy()` is only on the root `GUI` class.
Not on `GUIController`.